clients/src/main/java/org/apache/kafka/common/security/authenticator/AbstractLogin.java

```diff
@@ -58,14 +60,14 @@ public LoginContext login() throws LoginException {
         if (jaasConfigFile == null) {
             log.debug("System property '" + JaasUtils.JAVA_LOGIN_CONFIG_PARAM + "' is not set, using default JAAS configuration.");
         }
-        AppConfigurationEntry[] configEntries = Configuration.getConfiguration().getAppConfigurationEntry(loginContextName);
+        AppConfigurationEntry[] configEntries = jaasConfig.getAppConfigurationEntry(loginContextName);
         if (configEntries == null) {
             String errorMessage = "Could not find a '" + loginContextName + "' entry in the JAAS configuration. System property '" +
                 JaasUtils.JAVA_LOGIN_CONFIG_PARAM + "' is " + (jaasConfigFile == null ? "not set" : jaasConfigFile);
```

Comment:
Again, we need to revise these messages now that there's another way to set the jaas config.